### PR TITLE
54 show tracking number

### PR DESCRIPTION
**Pull request by jmcarp**
7ec1tjfi9s

On comment submit:
* Generate random submission ID
* Generate temporary URL for submission metadata
* Bind metadata ID to `publish_metadata` task
* Submit comment to regs.gov using `submit_comment` task
* Chain `submit_comment` to `publish_metadata` task
* Poll S3 key from client until it's not a 404

cc @cmc333333 @vrajmohan

### CODE REVIEW

File: regulations/management/commands/setup_cors.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def handle(self, *args, **options):
             CORSConfiguration={
                 'CORSRules': [
                     {
-                        'AllowedMethods': ['PUT'],
+                        'AllowedMethods': ['GET', 'PUT'],
```
Comment (cmc333333) [inline regulations/management/commands/setup_cors.py]: Would it make sense to have a separate bucket for the two types of content? I'm a bit apprehensive about allowing GET access to partially-uploaded docs.

Comment (cmc333333) [inline regulations/management/commands/setup_cors.py]: On second thought, the permissions granted below make a lot more sense than firewalling by bucket. Nevermind!

File: regulations/views/comment.py [hunk 2]
```diff
@@ -56,5 +52,21 @@ def preview_comment(request):
...
-    return JsonResponse({'status': 'submitted'})
+    s3 = tasks.make_s3_client()
+    metadata_key = get_random_string(50)
+    metadata_url = s3.generate_presigned_url(
+        ClientMethod='get_object',
+        Params={
+            'Bucket': settings.ATTACHMENT_BUCKET,
+            'Key': metadata_key,
+        },
+    )
```
Comment (cmc333333) [inline regulations/views/comment.py]: Do you know if this URL will have an expiry time?

Comment (jmcarp) [inline regulations/views/comment.py]: Good point--the expiration defaults to an hour from this call, but can be set to expire at any timestamp. We can bump up this time, and/or make it configurable.

File: regulations/static/regulations/js/source/views/comment/comment-review-view.js [hunk 3]
```diff
@@ -121,11 +121,33 @@ var CommentReviewView = Backbone.View.extend({
...
+      5000
+    );
+  },
+
+  setTrackingNumber: function(number) {
+    this.$status.html(
+      '<div>Comment submitted</div>' +
+      '<div>Tracking number: ' +
+        '<a href="http://www.regulations.gov/#!searchResults;rpp=25;po=0;s=' + number + '">' + number + '</a>' +
+      '</div>'
```
Comment (cmc333333) [inline regulations/static/regulations/js/source/views/comment/comment-review-view.js]: I'd vote for this markup to come from the backend in a later revision.

**Comment by cmc333333** [comment]
This is very cool. I'll merge shortly unless @vrajmohan has feedback

**Comment by vrajmohan** [comment]
:+1:

**Pull request merged**